app/ui-message/client/popup/messagePopupConfig.html

```diff
@@ -5,5 +5,9 @@
 		{{> messagePopup popupSlashCommandsConfig}}
 		{{> messagePopup popupEmojiConfig}}
 		{{> messagePopup popupReactionEmojiConfig}}
+
```

Comment:
why not migrate all messagePopup to the same behavior ?

Reply:
@ggazzo I believe it's more readable that way.